mri/operators/linear/wavelet.py

```diff
@@ -67,7 +67,7 @@ def __init__(self, wavelet_name, nb_scale=4, verbose=0, dim=2,
         if n_proc < 0:
             n_proc = joblib.cpu_count() + self.n_jobs + 1
         # Create transform queue for parallel execution
-        for i in range(min(n_proc, self.n_coils)):
+        for i in range(min(n_proc, self.n_coils)+1):
```

Comment:
what is this?

Reply:
No , I seem to have minute issues with travis , I was getting some `None` for some wavelets. This was just a way to test that, I am debugging travis. Removing this.